kapacitor/ast.go

```diff
@@ -851,3 +851,51 @@ func extractExec(node *pipeline.AlertNode, rule *chronograf.AlertRule) {
 
 	rule.AlertNodes = append(rule.AlertNodes, alert)
 }
+
+func extractPushover(node *pipeline.AlertNode, rule *chronograf.AlertRule) {
+	if node.PushoverHandlers == nil {
+		return
+	}
+	rule.Alerts = append(rule.Alerts, "pushover")
+	a := node.PushoverHandlers[0]
```

Comment:
I think it's because as of right now we're only supporting a single alert per alert node, but this is basically copied from the other `extract` functions on this page. I imagine changing this would be part of a larger refactor, when we start supporting multiple alerts per node in the server and client (which also doesn't support multiple currently).